README.md

```diff
@@ -6,45 +6,45 @@
 
 # Make your own fork of this app in Codespaces
 
-Step 1:  Click the "Open in Github Codespaces" button above and then click Fork!
+Step 1:  Right click the "Open in Github Codespaces" button above, open the link in a new tab and then click Fork!
```

Comment:
I did this and then lost my page and had to recreate it :(

Reply:
Ooh, good call -- I'll update the link to open in a new tab instead